cnxpublishing/views/admin.py

```diff
@@ -271,3 +295,250 @@ def admin_edit_site_message_POST(request):
+                if current_recipe != latest_recipe:
+                    state += ' stale_recipe'
+                state_icon = result.state
+                if state[:7] == "SUCCESS" and len(state) > 7:
+                    state_icon = 'PENDING'
+                states.append({
+                    'title': row[0].decode('utf-8'),
+                    'authors': format_autors(row[1]),
+                    'uuid': row[2],
+                    'print_style': row[3],
+                    'recipe': row[4],
+                    'created': row[-2],
+                    'state': state,
+                    'state_message': message,
+                    'state_icon': STATE_ICONS[state_icon]['class'],
+                    'state_icon_style': STATE_ICONS[state_icon]['style'],
+                    'status_link': request.route_path(
+                        'admin-content-status-single', uuid=row[2]),
+                    'content_link': request.route_path(
+                        'get-content', ident_hash=row[2])
+                })
+    status_filters = [request.GET.get('pending_filter', ''),
+                      request.GET.get('started_filter', ''),
+                      request.GET.get('retry_filter', ''),
+                      request.GET.get('failure_filter', ''),
+                      request.GET.get('success_filter', '')]
+    status_filters = [x for x in status_filters if x != '']
+    if status_filters == []:
+        status_filters = ["PENDING", "STARTED", "RETRY", "FAILURE", "SUCCESS"]
+    final_states = []
+    for state in states:
+        if state['state'].split(' ')[0] in status_filters:
+            final_states.append(state)
+
+    sort = request.GET.get('sort', 'bpsa.created DESC')
+    sort_match = SORTS_DICT[sort.split(' ')[0]]
+    sort_arrow = ARROW_MATCH[sort.split(' ')[1]]
+    if sort == "STATE ASC":
+        final_states = sorted(final_states, key=lambda x: x['state'])
+    if sort == "STATE DESC":
+        final_states = sorted(final_states,
+                              key=lambda x: x['state'], reverse=True)
+
+    num_entries = request.GET.get('number', 100)
+    page = request.GET.get('page', 1)
+    try:
+        page = int(page)
+        num_entries = int(num_entries)
+        start_entry = (page - 1) * num_entries
+    except ValueError:
+        raise httpexceptions.HTTPBadRequest(
+            'invalid page({}) or entries per page({})'.
+            format(page, num_entries))
+    final_states = final_states[start_entry: start_entry + num_entries]
+
+    return_args = {'start_entry': start_entry,
+                   'num_entries': num_entries,
+                   'page': page,
+                   'total_entries': len(final_states),
+                   'states': final_states,
+                   'sort_' + sort_match: sort_arrow,
+                   'sort': sort}
+    for f in (status_filters):
+        return_args[f] = "checked"
+    return_args.update(sql_args)
+    return return_args
```

Comment:
Sorry to be picky about this but args isn't synonymous with vars. Arguments are typically passed into functions rather than returned from them. From wiktionary, "A value, or reference to a value, passed to a function." Maybe the generic `results` is a better name for this type of value?